fairseq/search.py

```diff
@@ -46,10 +49,35 @@ def step(self, step, lprobs, scores):
     def set_src_lengths(self, src_lengths):
         self.src_lengths = src_lengths
 
+    @torch.jit.export
+    def init_constraints(self, batch_constraints: Optional[Tensor], beam_size: int):
+        """Initialize constraint states for constrained decoding (if supported).
+
+        :param batch_constraints:
+        :param beam_size:
```

Comment:
Please switch to the Google Python docstring convention (i.e., "Args:"), which we use this in most places in fairseq: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods